TLM/TLM/UI/TrafficManagerTool.cs

```diff
@@ -415,50 +415,81 @@ public class TrafficManagerTool
             DrawNodeCircle(cameraInfo, nodeId, GetToolColor(warning, false), alpha);
         }
 
+        /// <returns>the average half width of all connected segments</returns>
+        private float NodeRadius(ushort nodeId) {
```

Comment:
General idea function names should begin with a verb to denote an action done, i.e. GetNodeRadius(), while variables or properties can remain as nouns.